Add loadRows to SND table QUSes

labkey-martyp opened the pull request:
#### Rationale
Related PR in dataintegration calls loadRows instead of mergeRows in TransformTask.appendToTargetQuery.  Add loadRows to SND tables to call mergeRows or importRows depending on insert option.

#### Related Pull Requests
* https://github.com/LabKey/dataintegration/pull/99

#### Changes
* Add loadRows to SND table QUSes

labkey-martyp committed:
Add override for loadRows

labkey-martyp committed:
Add loadRows to SND table QUSes

labkey-jeckels reviewed (changes requested):
I take it that the loadRows() being inherited from AbstractQueryUpdateService isn't working here? A comment about what's wrong with it would be nice.

Also, with 5 duplicate copies of the overridden method, what about creating an intermediate superclass they can all inherit from (perhaps with the exception of AttributeDataTable which doesn't subclass SimpleTable)?

labkey-martyp committed:
Create SNDQueryUpdateService with common loadRows function

labkey-martyp commented:
@labkey-jeckels Good call.  All those QUSes were extending SimpleQueryUpdateService so I made a super class extending that and added the loadRows there.  Added comment there as well.
